src/utilities/buildClientSchema.js

```diff
@@ -253,9 +253,17 @@ export function buildClientSchema(
   function buildInterfaceDef(
     interfaceIntrospection: IntrospectionInterfaceType,
   ): GraphQLInterfaceType {
+    if (!interfaceIntrospection.interfaces) {
+      throw new Error(
+        'Introspection result missing interfaces: ' +
+          inspect(interfaceIntrospection),
+      );
+    }
+
```

Comment:
Please extract it into a separate function and reuse for building object type, similar to `buildFieldDefMap`

Reply:
What part here is reusable? Do you just want to consolidate the creation of this object?

```js
{
  name: interfaceIntrospection.name,
  description: interfaceIntrospection.description,
  interfaces: () => interfaceIntrospection.interfaces.map(getInterfaceType),
  fields: () => buildFieldDefMap(interfaceIntrospection),
}
```

If so, I'm not sure the extra redirection here is worth the DRY...

Reply:
@mike-marcacci I'm referring to:
```
    if (!interfaceIntrospection.interfaces) {
      throw new Error(
        'Introspection result missing interfaces: ' +
          inspect(interfaceIntrospection),
      );
    }
    return interfaceIntrospection.interfaces.map(getInterfaceType)
```

Reply:
Ah got it; good call. Fixed.